vendor/vendor.json

```diff
@@ -383,12 +383,6 @@
 			"version": "=v1.5.4",
 			"versionExact": "v1.5.4"
 		},
-		{
-			"checksumSHA1": "FhLvgtYfuKY0ow9wtLJRoeg7d6w=",
-			"path": "github.com/gogo/protobuf/gogoproto",
```

Comment:
Why is this getting removed?

Reply:
@brian-brazil details for that are in the commit (https://github.com/prometheus/prometheus/pull/4226/commits/a7528730306bff87d60cb538a5db54c5e37f4126) TLDR; the script that codegens out the proto files changed to not include that anymore, so I removed it. If this *should* be in use, then there is a bug in `genproto.sh` as it quite literally seds it out (https://github.com/prometheus/prometheus/blob/master/scripts/genproto.sh#L43)

Reply:
@brian-brazil bump :)

Reply:
Usually I would have already merged your change as your explanation makes sense, however we're still in moratorium.

Reply:
Oh, didn't realize there was a moratorium. So if we're all set here from a code perspective then I'll just wait for merge. I mostly wanted to make sure there wasn't something outstanding to do on this :)